src/api/forms/components/FormRow.vue

```diff
@@ -131,7 +131,12 @@ export default {
                     && data.value !== '';
             }
 
-            if (this.row.required && !valid) {
+            if (this.row.required && !valid
+                    || data.min < 0
```

Comment:
I have just sow image of the gauges and they have negative minimum value, so to fix this just delete `data.min < 0` to fix that problem of disabled "OK" button.